layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend

```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```

Comment:
How is this broken? I remember this being fixed a while ago: https://github.com/resin-os/meta-resin/commit/1d7061075e28c15572cda65a7c841eaadb92ab55

Did you have any problems with this?

Reply:
Still broken